layouts/default.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+<div>
+    <TheHeader @sidenavToggle="displaySidenav = !displaySidenav" />
+    <TheSidenav 
+        :show="displaySidenav"
+        @close="displaySidenav = false" />
+
+    <nuxt/>
+</div>
+</template>
+
+<script>
+import TheHeader from '@/components/Navigation/TheHeader'
+import TheSidenav from '@/components/Navigation/TheSidenav'
```

Comment:
We should include the file extension whenever possible.
```suggestion
import TheHeader from '@/components/Navigation/TheHeader.vue'
import TheSidenav from '@/components/Navigation/TheSidenav.vue'
```